apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V26__Add_fields_to_BeneficiariesInpatient.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * The column doesn't have a default value to avoid updating the column on migration. The pipeline server
+ * will populate the column as new beneficaries are added or existing beneficaries are updated. 
+ */
+
+alter table "Beneficiaries" add column "derivedMailingAddress1" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress2" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress3" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress4" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress5" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress6" varchar(40);
+
+alter table "Beneficiaries" add column "derivedCityName" varchar(100);
+
+alter table "Beneficiaries" add column "derivedStateCode" varchar(2);
```

Comment:
How does the data in this field differ from `"Beneficiaries"."stateCode"`?

Reply:
answer above..